perfmap.go

```diff
@@ -127,7 +132,8 @@ func (m *PerfMap) Start() error {
 	// Create and start the perf map
 	var err error
 	opt := perf.ReaderOptions{
-		Watermark: m.Watermark,
+		Watermark:    m.Watermark,
+		WakeupEvents: m.WakeupEvents,
```

Comment:
Should we just set one of these, to avoid confusions around which one gets applied by the kernel?

Reply:
Upstream handles if they are both set to non-zero values with an error https://github.com/cilium/ebpf/blob/f2b2f6d61aaf1fa8c0eb14c2a58f7e199fbd6086/perf/reader.go#L198-L200